### PR TITLE
[bitnami/tomcat] TOMCAT_EXTRA_JAVA_OPTS are added multiple times to tomcat-env.sh #53082

### PR DESCRIPTION
**Pull request by christian-scheible**
Preventing `TOMCAT_EXTRA_JAVA_OPTS` to be added multiple times to _tomcat-env.sh_ when the container is restarted.

Signed-off-by: Christian Scheible <christian.scheible@uni-konstanz.de>

### Description of the change

This changes checks if the additional **TOMCAT_EXTRA_JAVA_OPTS** have allready been added to _tomcat-env.sh_ before adding them. Otherwise every restart will add them again to the file.

### Benefits

This prevents adding options multiple times to the **JAVA_OPTS** which fails for example if someone want's to attach a debugger like this: `-agentlib:jdwp=transport=dt_socket,address=*:9543,suspend=n,server=y`

### Possible drawbacks

-

### Applicable issues

#53082

### Additional information

-

### CODE REVIEW

**Review by andresbono — COMMENTED** [review]
Thank you for your contribution! I think it makes sense. Could you please take a look at my comment?

File: bitnami/tomcat/10.1/debian-11/rootfs/opt/bitnami/scripts/libtomcat.sh [hunk 1]
```diff
@@ -163,11 +163,16 @@ tomcat_enable_application() {
 #########################
 tomcat_initialize() {
     if ! is_empty_value "$TOMCAT_EXTRA_JAVA_OPTS"; then
-        cat >>"${TOMCAT_BIN_DIR}/setenv.sh" <<EOF
+      # Check if $TOMCAT_EXTRA_JAVA_OPTS has already been added to setenv.sh
```
[the remark this answers is not in the document]
Comment (christian-scheible) [reply bitnami/tomcat/10.1/debian-11/rootfs/opt/bitnami/scripts/libtomcat.sh]: Hi it seems that I have now broken it by rebasing the branch. Should I start a new branch an a new pull request?

[the remark this answers is not in the document]
Comment (andresbono) [reply bitnami/tomcat/10.1/debian-11/rootfs/opt/bitnami/scripts/libtomcat.sh]: Hi, that is fine, you can open a new PR if you prefer in order to restore the changes.

[the remark this answers is not in the document]
Comment (christian-scheible) [reply bitnami/tomcat/10.1/debian-11/rootfs/opt/bitnami/scripts/libtomcat.sh]: Ok I made a new one: https://github.com/bitnami/containers/pull/53780

**Pull request closed without merging**